lib/bintray_client.rb

```diff
@@ -0,0 +1,67 @@
+require_relative './json_http_client.rb'
+
+##
+# +BintrayClient+ interfaces with the Bintray REST API
+#
+# @see https://www.jfrog.com/confluence/display/BT/Bintray+REST+API
+class BintrayClient
+
+  ##
+  # A new instance of +BintrayClient+
+  #
+  # @param user [String] the username to use to authenticate with the Bintray API
```

Comment:
Also, some of the examples [in the docs](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md) have a new line between the method description and the list of tags, while others don't. I decided to always add one because I seemed more readable like that.